.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
+    "test": {
+      "plugins": ["@babel/plugin-transform-modules-commonjs"]
+    }
+  }
```

Comment:
- [ ] Your tests are failing because (as indicated in the message) jest is unable to resolve the files. You forgot to add the closing parentheses here. Kindly do that to fix your tests issue.

  ```
  {
    "env": {
      "test": {
        "plugins": ["@babel/plugin-transform-modules-commonjs"]
      }
    }
  }
  ```
  Comments count test | Items count test
  :---:|:---:
  ![ezekiel-comments-test](https://user-images.githubusercontent.com/36542986/163364643-43bcda25-af3d-4181-951c-0f5a3485c647.jpg) | ![ezekiel-items-test](https://user-images.githubusercontent.com/36542986/163364693-2163ea44-72e8-4f5c-a0c7-3fb5da1f675e.jpg)

Reply:
Thank you.